Library/Homebrew/rubocops/class_cop.rb

```diff
@@ -82,6 +82,14 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
             return
           end
 
+          if test.body.single_line? &&
+             test.body.source.to_s.include?("version") ||
+             test.body.source.to_s.include?("help")
```

Comment:
Maybe also `-v` and `-h`?

Reply:
The `version` here is intended mostly to match `version.to_s` and `/#{version}/`. I wasn't sure on `-v` because sometimes people seem to map it to `--verbose`, which could have legitimate usage in a test, but I'm happy to go for it and see how things go if you/others don't think it unwise.

`-h` definitely sounds like a 👍.

Reply:
This will make `subversion` and `whelp` both raise an error. Can you make it a bit more specific to those cases?

Reply:
You can match for `\bversion\b`.